charts/psmdb-operator/values.yaml

```diff
@@ -40,11 +40,17 @@ serviceAccount:
 # annotations to add to the operator deployment
 annotations: {}
 
+# labels to add to the operator deployment
+labels: {}
+
 # annotations to add to the operator pod
 podAnnotations: {}
   # prometheus.io/scrape: "true"
   # prometheus.io/port: "8080"
 
+#labels to the operator pod
+podLabels: {}
```

Comment:
Please add these new options to README file.

Reply:
fixed